frontend/test/metabase/scenarios/dashboard-filters/reproductions/22197-reload-card-without-change.cy.spec.js

```diff
@@ -35,19 +35,23 @@ describe("issue 22197", () => {
   it("should not reload cards in a dashboard when corresponding filters are not changed (metabase#22197)", () => {
     createQuestionsAndDashboard().then(
       ({ dashboard_id, card1_id, card2_id }) => {
-        setFilterMapping({ dashboard_id, card1_id, card2_id });
         interceptRequests({ dashboard_id, card1_id, card2_id });
+        setFilterMapping({ dashboard_id, card1_id, card2_id }).then(() => {
```

Comment:
I think there's a race condition in this e2e test (it broke randomly, though it might seem related), so I've sequenced these promises and corrected the logic to open and fill the parameter with a value.